### PR TITLE
[circle2circle]Introduce RemoveRedundantTranspose pass in circle2circle

### PR DESCRIPTION
**Pull request by struss**
Parent Issue : #5062
Draft : #5074

This commit introduce `RemoveRedundantTranspose` pass in `circle2circle`

ONE-DCO-1.0-Signed-off-by: KiDeuk Bang <rrstrous@nate.com>

### CODE REVIEW

**Review by jinevening — DISMISSED** [review]
LGTM

File: compiler/circle2circle/src/Circle2Circle.cpp [hunk 1]
```diff
@@ -220,6 +226,8 @@ int entry(int argc, char **argv)
     options->enable(Algorithms::MakeBatchNormGammaPositive);
   if (arser.get<bool>("--fuse_preactivation_batchnorm"))
     options->enable(Algorithms::FusePreActivationBatchNorm);
+  if (arser.get<bool>("--remove_redundant_transpose"))
+    options->enable(Algorithms::RemoveRedundantTranspose);
```
Comment (seanshpark) [inline compiler/circle2circle/src/Circle2Circle.cpp:230]: was this added?

**Review by seanshpark — APPROVED** [review]
LGTM

File: compiler/circle2circle/src/Circle2Circle.cpp [hunk 2]
```diff
@@ -110,6 +110,12 @@ int entry(int argc, char **argv)
       .default_value(false)
       .help("This will fuse BatchNorm operators of pre-activations to Convolution operator");
 
+  arser.add_argument("--remove_redundant_transpose")
+      .nargs(0)
+      .required(false)
+      .default_value(false)
```
Comment (llFreetimell) [inline compiler/circle2circle/src/Circle2Circle.cpp:116]: How about setting default_value as `true` for this?

Comment (seanshpark) [reply compiler/circle2circle/src/Circle2Circle.cpp:116]: then need to add to `--all` too

Comment (mhs4670go) [reply compiler/circle2circle/src/Circle2Circle.cpp:116]: both gets same result. Just adding it to `--all` section is enough.

Comment (struss) [reply compiler/circle2circle/src/Circle2Circle.cpp:116]: Okay, I will add on `--all` section.

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by jinevening — APPROVED** [review]

**Review by llFreetimell — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**